lib.rs

```diff
@@ -87,16 +140,24 @@ impl<D: Decorator> Format<D> {
         let r_decorator = self.decorator.decorate(record);
 
 
-        try!(self.print_msg_header(io, &r_decorator, record));
+        let mut skip_msg_comma = try!(self.print_msg_header(io, &r_decorator, record));
```

Comment:
Intuition tells me, it should be possible, but I am not versed enough with lifetime details to be sure. Please let me know the details (error message) if it's not possible - maybe will figure it out.